bundles/oms-payone-error/src/FondOfKudu/Zed/OmsPayoneError/Communication/Plugin/Oms/Condition/Error3dSecureConditionPlugin.php

```diff
@@ -28,8 +28,8 @@ class Error3dSecureConditionPlugin extends AbstractPlugin implements ConditionIn
      */
     public function check(SpySalesOrderItem $orderItem): bool
     {
-        $errorCode = (int)$this->getRepository()->findPaymentPayoneApiLogErrorWithIdSalesOrder($orderItem->getFkSalesOrder());
+        $errorCode = $this->getRepository()->findPaymentPayoneApiLogErrorWithIdSalesOrder($orderItem->getFkSalesOrder());
 
-        return $errorCode >= static::ERROR_MIN && $errorCode <= static::ERROR_MAX;
+        return $errorCode !== null && (int)$errorCode >= static::ERROR_MIN && (int)$errorCode <= static::ERROR_MAX;
```

Comment:
Finde das hier etwas unübersichtlich könnte man nicht vorher ein return machen, wenn es null ist? 

```suggestion
        if ($errorCode === null) {
             return false;
        }
        
        return (int)$errorCode >= static::ERROR_MIN && (int)$errorCode <= static::ERROR_MAX;
```

Reply:
klar! :)